kms/api-client/snippets_test.py

```diff
@@ -177,31 +179,39 @@ def test_key_policy(self):
                                                  self.symId,
                                                  self.member,
                                                  self.role)
-        policy = snippets.get_crypto_key_policy(self.project_id,
-                                                self.location,
-                                                self.keyring_id,
-                                                self.symId)
-        found = False
-        for b in list(policy.bindings):
-            if b.role == self.role and self.member in b.members:
-                found = True
-        assert found
+
+        def check_policy():
+            policy = snippets.get_crypto_key_policy(self.project_id,
```

Comment:
Nit (optional), I tend to avoid hanging indent and the mixed camelCase and snake_case is a bit weird:

```
policy = snippets.get_crypto_key_policy(
    self.project_id,
    self.location,
    self.keyring_id,
    self.sym_id)
```

Reply:
good point, fixed